lib/repl.js

```diff
@@ -203,6 +204,9 @@ function REPLServer(prompt,
     }
   }
 
+  const preview = options.terminal &&
+    (options.preview !== undefined ? !!options.preview : true);
```

Comment:
Doesn't this still prevent previews from being enabled if `terminal` is false-y? Why not only take `.terminal` into account if `options.preview === undefined`?

Reply:
That is correct. I would have liked to do that but the current proposal does not support previews for non-terminals yet. That requires additional work and I would rather do that in a follow-up PR.